team-norms/retrospectives.md

```diff
@@ -0,0 +1,66 @@
+# Retrospectives
+
+These retrospectives occur every 2 weeks on a Friday. There are 2 hours booked for this purpose. 
+
+There are two elements to retrospectives:
+
+* Retrospective (smaller groups performing retrospectives)
+* Comradrospective (whole company retrospective format)
+
+## Purpose 
+
+The purpose of retrospectives is to:
+
+* Examine outcomes from previous retrospective, and understand what went well, and what didn’t.
+* Create action points or decide on problems that the team will tackle
+* Provide a conduit to allow continuous improvement to be facilitated safely within teams.
+
+
+The purpose of the comradrospective is to:
+
+*  Examine previous action points, what went well and what didn’t.
+*  Avoid “localised optimisations” i.e. the action points of one smaller group, hurting the whole.
+*  Set one or two action points, or problems, for the whole company to focus on.
+
+### Action points
+
+Remember that action points are only potential solutions to a problem. In order to aid examining your own achievements, being able to measure the impact of your action points by some form of measurable yard-stick can be quite helpful. Establishing a definition of awesome is one accepted solution to this, or some sort of hard metric can also help.
+
+### Tackling problems
+
+Sometimes it is not immediately obvious how to solve a problem that the team is collectively facing. In these situations it may be wise to have the team attempt to resolve a particularly pressing pain point during normal working time.
+
+Similar to action points, it’s important that the team is able to measure and sense check whether it has made progress towards some goal. 
+
+Make sure the team has awareness on what the outcome of potential solutions would look like.
+
+## Format
```

Comment:
How are we determining the topics covered in the retrospective / comradrospectives?